docs/cvl/builtin.md

```diff
@@ -1,10 +1,27 @@
 Built-in Rules
 ==============
+`Certora` Prover has built-in general-purpose rules targeted at finding known vulnerabilities.
+These rules can be verified on a contract out-of-the-box.
```

Comment:
Replace with
These rules can be verified through an on a contract out-of-the-box contract